static/.htaccess

```diff
@@ -113,6 +113,15 @@ RewriteRule ^docs/next/apis-tools/tasklist-api/(.*)$ /docs/next/apis-tools/camun
 #   8.7: content moves introduced prior to the release of version 8.7.
 #---------------------------------------------------------------------------------
 
+# Restructure announcements and release notes for 8.6
+RewriteRule ^docs/reference/announcements/announcements-850/?$ /docs/reference/announcements-release-notes/850/850-announcements/$1 [R=301,L]
+RewriteRule ^docs/reference/release-notes/850/?$ /docs/reference/announcements-release-notes/850/850-release-notes/$1 [R=301,L]
+RewriteRule ^docs/reference/announcements/announcements-860/?$ /docs/reference/announcements-release-notes/860/860-announcements/$1 [R=301,L]
+RewriteRule ^docs/reference/release-notes/860/?$ /docs/reference/announcements-release-notes/860/860-release-notes/$1 [R=301,L]
+RewriteRule ^docs/reference/release-policy/?$ /docs/reference/announcements-release-notes/release-policy/$1 [R=301,L]
+RewriteRule ^docs/reference/release-notes/?$ /docs/reference/announcements-release-notes/overview/$1 [R=301,L]
+RewriteRule ^docs/reference/announcements/?$ /docs/reference/announcements-release-notes/overview/$1 [R=301,L]
+
```

Comment:
@pepopowitz should these also include the versioned `/8.6/` URLs?

Reply:
If we follow existing practices, no. `/8.6/` URLs will currently redirect to `/` URLs, which will be caught by these rules. 

But I think there is a bigger conversation, about whether those existing practices should be changed now that we added the `/8.6/` -> `/` redirect. I'll start a conversation in Slack. I don't believe that conversation should block this PR, because if we change existing practices I'll have to go fill in all sorts of redirects, not just these few.

Reply:
@pepopowitz Merging this, but let me know if we need to go back through the redirects.